eng/pipelines/runtime.yml

```diff
@@ -474,6 +474,7 @@ jobs:
     buildConfig: Release
     runtimeFlavor: mono
     platforms:
+    - Android_x86
```

Comment:
This will duplicate jobs and we intentionally did this as a mix and match in between Debug and Release for PRs and CI to avoid having a ton of jobs. Please remove these additions and leave as is.